### PR TITLE
Feature/precinct splits

### PR DESCRIPTION
**Pull request by pstenbjorn**
Moving this to a merge into vip5 branch.

This addresses https://github.com/votinginfoproject/vip-specification/issues/27

### CODE REVIEW

**Comment by pstenbjorn** [comment]
@jungshadow I closed the other PR and this should be the correct branch.

File: precinct_splits.xsd [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    </xs:complexType>
+  </xs:element>
+  <xs:simpleType name="identifierType">
+  <xs:restriction base="xs:string">
+    <xs:enumeration value="ocd-id"/>
+    <xs:enumeration value="FIPS"/>
+    <xs:enumeration value="national"/>
+    <xs:enumeration value="state"/>
+    <xs:enumeration value="local"/>
+    <xs:enumeration value="OTHER"/>
```
Comment (jungshadow) [inline precinct_splits.xsd]: @pstenbjorn I know it's on me to create the coding style guide (working on it), but, following some of the 1622 conventions, which will start as the basis for the guide, the enumerations should be lowercase.

File: vip_spec.xsd [hunk 2]
```diff
@@ -364,13 +359,59 @@
...
+            <!-- The language defaults to English in this case. -->
+            <xs:simpleContent>
+                <xs:extension base="xs:string">
+                    <xs:attribute name="inter_text_id" type="xs:string"/>
+                </xs:extension>
+            </xs:simpleContent>
+        </xs:complexType>
+    </xs:union>
 </xs:complexType>
+<xs:complexType name="pollingLocationType">
```
Comment (jungshadow) [inline vip_spec.xsd]: @pstenbjorn Maybe I'm missing something (quite possible), but does it make sense to put the definition of `polling_location` here (i.e. couldn't this all be defined above)?

Comment (pstenbjorn) [inline vip_spec.xsd]: @jungshadow there isn't a reason aside from its being a bit of a legacy declaration.  I had been moving to complexType definitions and this one remained.  For consistency it should be declared as an element and I will change it.

File: precinct_splits.xsd (hunk 1 above)
Comment (pstenbjorn) [inline precinct_splits.xsd]: @jungshadow yes I will clean this up.

**Comment by pstenbjorn** [comment]
@jungshadow this commit https://github.com/votinginfoproject/vip-specification/commit/40a287877ddc72eeb6471260c10ff8cf911feec4  addresses syntax and some other consistency issues between branches.   As soon as PR from @cjerdonek is merged, I will make one more pass through this branch before merging.

**Comment by jungshadow** [comment]
@pstenbjorn I merged in the vip5 branch and fixed the conflicts that showed up then merged in external identifiers and the precinct changes. I'll let you double check the `vip_spec.xsd` to see if it's what you expect. Feel free to merge if it's good.

**Pull request merged**